scripts/pkg/windows/nri-amd64-installer/win_package.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+PLATFORM="x64"
+PFX_PATH='mycert.pfx'
+
+# Parse arguments
+for i in "$@"
+do
+case $i in
+    -exporterName=*)
+    INTEGRATION_NAME="${i#*=}"
+    ;;
+    -version=*)
+    INTEGRATION_VERSION="${i#*=}"
+    ;;
+    -upgradeGUID=*)
+    UPGRADE_CODE="${i#*=}"
+    ;;
+    -pfx_passphrase=*)
+    PFX_PASSPHRASE="${i#*=}"
+    ;;
+esac
+done
+
+# verify version number format
+if ! [[ $INTEGRATION_VERSION =~ ^[0-9]+((\.[0-9]+)){2}$ ]]; then
+  echo "-version must follow a numeric major.minor.patch semantic versioning schema (received: $INTEGRATION_VERSION)"
+  exit
+fi
+
+# Remove binaries
+rm -rf bin obj
+
+MSBuild="'$(find /c/Windows/Microsoft.NET/Framework64/ -path "*/v4*/MSBuild.exe")'"
+SIGN_TOOL="'$(find /c/Program*Files*86*/ -path "*/SignTool/signtool.exe")'"
+BINARIES_PATH="'..\..\..\..\exporters\\${INTEGRATION_NAME}\target\bin\windows_amd64\'"
+
+eval $MSBuild nri-installer.wixproj \
+    -p:IntegrationName=$INTEGRATION_NAME \
+    -p:Platform=$PLATFORM \
+    -p:IntegrationVersion=$INTEGRATION_VERSION \
+    -p:BinariesPath=$BINARIES_PATH \
+    -p:UpgradeCode=$UPGRADE_CODE
```

Comment:
elsewhere we are quoting variables, can we quote them here as well?

```
eval "$MSBuild" nri-installer.wixproj \
    -p:IntegrationName="$INTEGRATION_NAME" \

```

Reply:
<img width="953" alt="Screenshot 2022-12-12 at 15 52 59" src="https://user-images.githubusercontent.com/43335750/207076577-a6c0bf07-6628-48ca-aec5-231ff82bd029.png">